### PR TITLE
[stable/nginx-ingress]update rbac apiversion

### PR DESCRIPTION
**Pull request by zouyee**
/cc @mattfarina

### CODE REVIEW

File: stable/nginx-ingress/templates/clusterrole.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.rbac.create -}}
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (mattfarina) [inline stable/nginx-ingress/templates/clusterrole.yaml:2]: As I understand it, the charts repo supports Kubernetes version N and N-1. Right now that versions 1.8 and 1.7. So, the charts need to work with both versions. The 1.7 API supports v1beta1 and not v1. So, this should be merged when 1.9 comes out.

cc: @kubernetes/charts-maintainers

Comment (prydonius) [reply stable/nginx-ingress/templates/clusterrole.yaml:2]: +1 we should keep this as v1beta1 for now

Comment (stamak) [reply stable/nginx-ingress/templates/clusterrole.yaml:2]: actually chart supports ONLY N-1(1.7) k8s version
what to do with k8s version N(1.8)?

Comment (mattfarina) [reply stable/nginx-ingress/templates/clusterrole.yaml:2]: @stamak [According to the deprecation policy, the beta API will be around for 2 releases or 6 months](https://kubernetes.io/docs/reference/deprecation-policy/). So, the `v1beta1` API will still work for 1.8.

Comment (stamak) [reply stable/nginx-ingress/templates/clusterrole.yaml:2]: thx

**Review by mattfarina — CHANGES_REQUESTED** [review]
This PR can't be merged until after Kubernetes 1.9 is out. rbac/v1 is supported in Kubernetes 1.8 but not 1.7. The [charts repo needs to support the latest and the previous minor releases](https://github.com/kubernetes/charts/blob/master/README.md#supported-kubernetes-versions). [Beta APIs are supported for 2 versions/6 months after the v1 is out](https://kubernetes.io/docs/reference/deprecation-policy/) so we can continue to use beta1 until after 1.9 is out. [1.9 is slated to be released on Dec 13th](https://github.com/kubernetes/features/blob/master/release-1.9/release-1.9.md). At that time if you would like to contribute rbac/v1 support it would be appreciated.

Note, at that point the chart version will likely need to be changed from the version in the PR to be newer than the chart at that time.

**Comment by zouyee** [comment]
thank you for your response, @mattfarina

**Comment by viglesiasce** [comment]
Closing as stale.

**Pull request closed without merging**